src/crepe/facade/Texture.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+// FIXME: this header can't be included because this is an API header, and SDL2 development
```

Comment:
Deze fixme comment kan weg omdat Texture nu niet langer vanuit de API toegankelijk is